Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -468,6 +464,29 @@ private bool ShouldStartBrowser()
             return true;
         }
 
+        private string GetBrowserPath()
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+
+            var uiShellOpenDocument = (IVsUIShellOpenDocument)ServiceProvider.GlobalProvider.GetService(typeof(SVsUIShellOpenDocument));
+            uiShellOpenDocument.GetFirstDefaultPreviewer(out var defaultBrowserPath, out _, out _);
+
+            return defaultBrowserPath;
+        }
+
+        private static void OpenBrowser(string browserPath, string webBrowserUrl)
+        {
+            // Chrome has known issues with being launched as admin.
+            if (!string.IsNullOrEmpty(browserPath) && browserPath.EndsWith(NodejsConstants.ChromeApplicationName))
```

Comment:
I guess you do check here if the browserPath was set correctly, so the previous comment about not handling failure on the call to GetFirstDefaultPreviewer is likely benign.